content/blog/getting-started-with-volume-snapshots-on-a-kubernetes-cluster-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,840 @@
+
+The following shows the details about deployed HPE CSI driver for K8s in the cluster 
+to its namespace *hpe-storage*: 
+ 
+```markdown
+$ k﻿ubectl get all -n hpe-storage
+NAME                                       READY   STATUS    RESTARTS      AGE
+pod/hpe-csi-controller-54cf448d85-g4w4c    9/9     Running   0             56d
+pod/hpe-csi-node-5xtdb                     2/2     Running   0             56d
+pod/nimble-csp-74d57f9487-qxwln            1/1     Running   0             56d
+pod/primera3par-csp-59f5dfc499-hfghx       1/1     Running   0             56d
+pod/snapshot-controller-5fd799f6b5-f6k7n   1/1     Running   6 (22d ago)   56d
+pod/snapshot-controller-5fd799f6b5-z62dc   1/1     Running   2 (27d ago)   56d
+
+NAME                          TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+service/alletra6000-csp-svc   ClusterIP   10.101.79.85    <none>        8080/TCP   56d
+service/alletra9000-csp-svc   ClusterIP   10.97.147.230   <none>        8080/TCP   56d
+service/nimble-csp-svc        ClusterIP   10.110.238.43   <none>        8080/TCP   56d
+service/primera3par-csp-svc   ClusterIP   10.101.42.76    <none>        8080/TCP   56d
+
+NAME                          DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+daemonset.apps/hpe-csi-node   1         1         1       1            1           <none>          56d
+
+NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/hpe-csi-controller    1/1     1            1           56d
+deployment.apps/nimble-csp            1/1     1            1           56d
+deployment.apps/primera3par-csp       1/1     1            1           56d
+deployment.apps/snapshot-controller   2/2     2            2           56d
+
+NAME                                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/hpe-csi-controller-54cf448d85    1         1         1       56d
+replicaset.apps/nimble-csp-74d57f9487            1         1         1       56d
+replicaset.apps/primera3par-csp-59f5dfc499       1         1         1       56d
+replicaset.apps/snapshot-controller-5fd799f6b5   2         2         2       56d
+```
+
+
+
+
+As part of HPE CSI driver configuration, a list of _StorageClasses_ is created that refers to the CSI driver name. The _PersistentVolumeClaim_ (PVC) can then be created that uses the _StorageClass_ to dynamically provision PV backed by the HPE storage systems. Apart from features such as dynamic provisioning, raw block volumes, inline ephemeral volumes, and volume encryption, HPE CSI driver implements and supports volume snapshot on K8s cluster. As you can see in above deployment, the common snapshot controller _snapshot-controller_ and a _VolumeSnapshotClass_, together with a list of snapshot *CustomResourceDefinitions* (CRDs), get all deployed and added to the cluster. 
```

Comment:
The second sentence should be modified to:
The PersistentVolumeClaim (PVC) can then be created**, which** uses the StorageClass to dynamically provision **a** PV backed **up** by the HPE storage systems."

In the next sentence, add the word "a" prior to "K8s cluster."

In the final sentence, last line, modify the wording slightly to "**all get** deployed and..."